docs/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright 2020 Joe Drago. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+if(APPLE OR UNIX)
+    add_subdirectory(man)
+endif()
```

Comment:
Maybe better to use `NOT MSVC` as the conditional since `MINGW` still uses man pages, but they do not set `UNIX` to true

Reply:
Also, the `APPLE` part is redundant since `UNIX` is set when `APPLE` is also set